### ISSUE TITLE
Linting is failing but not listing any reason why

### ISSUE THREAD

**Issue by colin-mccarthy** [ansible/ansible-lint-action#13]
Lint is failing but not listing any reason why.

Tested with Ansible lint pre-commit hook and it passes. Seems like the container is failing. Saw this issue on multiple repos.

During the Lint Ansible Playbook step of the build it fails with this output.

`
/usr/bin/docker run --name e87b526347bc7600b44020817b6d7bf28f95d6_7c745d --label e87b52 --workdir /github/workspace --rm -e INPUT_TARGETS -e INPUT_ARGS -e HOME -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/ansible_csv/ansible_csv":"/github/workspace" e87b52:6347bc7600b44020817b6d7bf28f95d6
`

**Comment by webknjaz** [comment]
Show your workflow/links/anything else useful. Currently, I don't know what you're referring to.

**Comment by webknjaz** [comment]
Also, pre-commit hook is defined in ansible-lint repo, not in the action repo.

**Comment by colin-mccarthy** [comment]
I ran the pre-commit hook myself as a test.

Here is a link to one of the builds:
https://github.com/colin-mccarthy/ansible_lint_demo/runs/489028995?check_suite_focus=true

During the "Lint Ansible Playbook" step of the build it fails with out giving any reason why.

**Comment by webknjaz** [comment]
Right, so you copy-pasted an example from readme: https://github.com/colin-mccarthy/ansible_lint_demo/actions/runs/50443849/workflow. But you should also read the comments there explaining what to set in the "with:" section.

We just run `ansible-lint` with whatever you specify.

Action item for this project: try to validate inputs and yell at empty ones.

**Comment by colin-mccarthy** [comment]
Are you suggesting I need to alter the targets parameter?

**Comment by webknjaz** [comment]
Yes https://github.com/ansible/ansible-lint-action/issues/12#issuecomment-591986622

**Comment by colin-mccarthy** [comment]
Updating the targets section did not resolve the issue:

https://github.com/colin-mccarthy/ansible_lint_demo/runs/490265426?check_suite_focus=true

**Comment by webknjaz** [comment]
Try updating target ref to `v4.1.0.post0` or `master` and see what happens.

**Comment by webknjaz** [comment]
Or pin to sha `151b9a2` (the safest option security-wise)

**Comment by colin-mccarthy** [comment]
Okay working now with:

`uses: ansible/ansible-lint-action@151b9a2`

`targets: "./"`

Thanks for clarifying the parameters!

**Comment by webknjaz** [comment]
*/me needs to finally cut another tag*

**Comment by nleiva** [comment]
Is this definitive? In my scenario,  `"./"` doesn't match any file and the test shows as successful, which is misleading. On the other hand `"*.yml"` seems to work fine, it translates that to the actual filenames (matches all the files in the root and subfolders (not the folder `tests` though).

Not working [with "./"](https://github.com/nleiva/ansible-snow/runs/812553833)

```bash
+ ansible-lint -v --force-color ./
+ shopt -u globstar
```

 Working [with "*.yml"](https://github.com/nleiva/ansible-snow/runs/819664986)

```bash
+ ansible-lint -v --force-color add-to-inventory.yml create-incident.yml inventory-snow.yml ios-genie-show-ver.yml ios-show-ver.yml modify-incident.yml request-status.yml
/github/workspace /github/workspace
Examining add-to-inventory.yml of type playbook
...
+ shopt -u globstar
```

**Comment by webknjaz** [comment]
@nleiva master doesn't pin the ansible-lint version in Dockerfile (which is build at the beginning of each GHA workflow and is not reused from cache). The args are passed to `ansible-lint` that has its own autodiscovery. There's been some bugs with it so I don't know if the version you hit ([4.2.0](https://github.com/nleiva/ansible-snow/runs/812553833#step:2:411)) works as you want. It'll probably get fixed with the release of v4.3.0 later this summer.

**Comment by nleiva** [comment]
Oh, `ansible-lint` has a bug, got it. Thank you for taking a look at my logs. 

I went with `@master` as `@4.1.0` requires an env variable (`ACTION_PLAYBOOK_NAME`). I prefer the newer version. I'll keep an eyes on the updates.
